.github/workflows/deployHelpSite.yml

```diff
@@ -11,6 +11,12 @@ jobs:
     steps:
       - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
 
+      - name: Get merged pull request
+        id: getMergedPullRequest
+        uses: roryabraham/action-get-merged-pull-request@7a7a194f6ff8f3eef58c822083695a97314ebec1
```

Comment:
We could do that ... I was hoping that this would be a temporary fix but it seems like the upstream was abandoned. We can probably just ditch this action and instead use the `gh` cli directly. I've been slowly migrating things over from community actions to the GitHub CLI on your recommendation for a while now.